pkgs/by-name/za/zarf/package.nix

```
@@ -4,40 +4,69 @@
   fetchFromGitHub,
   installShellFiles,
   stdenv,
+  git,
 }:
 
+let
+  # Convert UNIX timestamp to ISO 8601 format (Go's expected format)
+  buildDate = builtins.toString builtins.currentTime;
+
+in
 buildGoModule rec {
   pname = "zarf";
-  version = "0.41.0";
+  version = "0.50.0";
 
   src = fetchFromGitHub {
-    owner = "defenseunicorns";
+    owner = "zarf-dev";
     repo = "zarf";
     rev = "v${version}";
```

```suggestion
    tag = "v${version}"; # also update ldflags
```
This guarantees a tag is fetched, not a branch